core/app/[locale]/layout.tsx

```diff
@@ -92,12 +92,10 @@ export default async function RootLayout({ params, children }: Props) {
   // https://next-intl-docs.vercel.app/docs/getting-started/app-router#add-setRequestLocale-to-all-layouts-and-pages
   setRequestLocale(locale);
 
-  const messages = await getMessages();
-
   return (
     <html className={clsx(fonts.map((f) => f.variable))} lang={locale}>
       <body>
-        <NextIntlClientProvider locale={locale} messages={messages}>
+        <NextIntlClientProvider>
```

Comment:
`next-intl` now inherits these values when `NextIntlClientProvider` is rendered in a server component. Read more in the docs: https://next-intl.dev/docs/usage/configuration#nextintlclientprovider